docs/contributing/999-rfcs/2024-06-26-wing-docs-v2.md

```diff
@@ -0,0 +1,161 @@
+
+**Contributor** - Open source contributors to wing. They understand the basics of the cloud and are looking for ways to contribute ideas, code and resources to wing.
+
+|  | Full stack developer | Platform engineer | Contributor |
+| --- | --- | --- | --- |
+| How do they use wing? | Building applications using the wing framework. | Building infrastructure for internal teams through custom platforms or wing libs. | Contributing to wing  |
+| How do they add business value? | Wants more focus on business logic and applications rather than worry about the infrastructure underneath. | Wants to enable developers, avoid being bottlenecks and focus on business agility. They do this by abstracting away complexity, maximising automation and give developers a self service developer experince. |  |
+| Level of understanding | May not understand the basics of the cloud (Queues, Functions), needs to be explained. | Understands the basics of the cloud (Queues, Functions), does not need this explained to them. | Understands the basics of the cloud (Queues, Functions), does not need this explained to them. |
+| How do they learn? | Benefits from how-to guides and examples. How to guides include things like deploying websites, deploying resources into the cloud, focused on business logic. | Benefits from how-to guides and examples. More focused on how to create and maintain libs that others can use or examples of creating custom platforms. | Benefits from how-to guides, either deploying applications into the cloud or creating libs and platforms for others. |
+| How do they interact with wing? | Consumer of wing standard libs, custom wing libs and custom platforms. | Produces wing libs and custom platforms for others to consume. | Can be both the consumer of wing and author of libs. |
+| Testing | Needs to understand how to test wing applications | Needs to understand how to test wing applications and libs. | Needs to understand how to test wing applications and libs. |
+| Programming langugages | May be interested in multi-lang preflight syntax | May be interested in multi-lang preflight syntax  | May be interested in multi-lang preflight syntax  |
+| What level of understanding do they need with wing? | Needs to understand the core concepts of wing and may need to the ability to dive deeper into wing. | Needs to understand the core concepts of wing and deeper dive content. | Needs to understand the core concepts of wing and deeper dive content
+
+Each of these personas have a different need for wing documentation and may have different entry points within the docs. Based on these personas, the proposed solution is to organise existing content, introduce new sections/pages, add more automation into docs and develop a new patterns/examples page.
+
+## Sitemap proposal
+
+The sitemap below visually explains the proposed solution for docs v2. 
+
+
+[Open the sitemap](https://drive.google.com/file/d/1jsZ4cXpf2v8Au3qPTJsU50PtK0zV5rqZ/view?usp=sharing)
```

Comment:
Where do you think something like our [compatibility matrix](https://www.winglang.io/docs/standard-library/compatibility-matrix) ought to go in the site map? This page / concept might also need a refresh in some way, not sure

Reply:
Yeah I think for now I put it under the `/API` route in the site map. Although I do wonder if there could also be value putting these in the platform pages, are at least a page to this.